zerver/migrations/0501_remove_realm_upload_quota_gb.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.10 on 2024-03-10 04:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0500_realm_zulip_update_announcements_stream"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="realm",
+            name="upload_quota_gb",
+        ),
+        migrations.AddField(
+            model_name="realm",
+            name="custom_upload_quota_gb",
+            field=models.IntegerField(null=True),
+        ),
+    ]
```

Comment:
I think it might be worth avoiding the migration by using https://docs.djangoproject.com/en/5.0/ref/models/fields/#db-column to keep the DB column named `upload_quota_gb`.

Reply:
@timabbott Hmm, but I imagine we'll still need to reset the value of this column to NULL for basically every row - since we want most organization to have the limit be calculated based on plan, via the `@property`, rather than set up as a custom limit in the database.

So we'll still need a migration, just a different one. Is that better than drop+create new column? We might even need to migrate with batching if we go with the above instead and have to update all the rows?

Reply:
Ahh, yeah, if that's what we're doing, your drop+create plan seems fine. Let's just make sure to add the new column in a commit earlier than dropping the old column, so it's easy to deploy with that last migration disabled.